CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ option(WITH_C_API       "Compile PaddlePaddle with C-API(Prediction)"   OFF)
 option(WITH_GOLANG      "Compile PaddlePaddle with GOLANG"              OFF)
 option(GLIDE_INSTALL    "Download and install go dependencies "         ON)
 option(USE_NNPACK       "Compile PaddlePaddle with NNPACK library"      OFF)
+option(UNITTEST_USE_VIRTUALENV "Python unittest with virtualenv"        ON)
```

Comment:
This PR is not to use Python `virtualenv` in Docker. Because in docker, Python packages are managed by Docker. 

Not use `virtualenv` in the unit tests will faster unit tests in docker.